src/WPF/ArcGISRuntime.WPF.Viewer/ArcGISLocalServer.AGSDeployment

```diff
@@ -1,109 +0,0 @@
-              <Package name="Dameng" enabled="false" />
-              <!--Adds support for direct connect to geodatabases stored in DB2.-->
-              <!--DB2 Drivers must be present on end users machine.-->
-              <Package name="DB2" enabled="false" />
-              <!--Adds support for direct connect to geodatabases stored in Informix.-->
-              <!--Informix Drivers must be present on end users machine.-->
-              <Package name="Informix" enabled="false" />
-              <!--Adds support for direct connect to geodatabases stored in Netezza.-->
-              <!--Netezza Drivers must be present on end users machine.-->
-              <Package name="Netezza" enabled="false" />
-              <!-- Adds support for direct connect to geodatabases stored in Oracle.-->
-              <!--Oracle Drivers must be present on end users machine.-->
-              <Package name="Oracle" enabled="false" />
-              <!--Adds support for direct connect to geodatabases stored in PostgreSQL.-->
-              <!--PostgreSQL Drivers must be present on end users machine.-->
-              <Package name="PostgreSQL" enabled="false" />
-              <!--Adds support for direct connect to geodatabases stored in SAP HANA.-->
-              <!--SAP HANA SQL Server Drivers must be present on end users machine.-->
-              <Package name="SAP HANA" enabled="false" />
-              <!--Adds support for direct connect to geodatabases stored in SQL Server.-->
-              <!--MS SQL Server Drivers must be present on end users machine.-->
-              <Package name="SQL Server" enabled="false" />
-              <!--Adds support for direct connect to geodatabases stored in SQL Server.-->
-              <!--MS SQL Server Drivers must be present on end users machine.-->
-              <Package name="SQLite" enabled="false">
-                <ChildPackages>
-                  <!--Adds support for SpatiaLite geometry type-->
-                  <Package name="SpatiaLite" enabled="false" />
-                </ChildPackages>
-              </Package>
-              <!--Adds support for direct connect to geodatabases stored in Teradata.-->
-              <!--Teradata Drivers must be present on end users machine.-->
-              <Package name="Teradata" enabled="false" />
-            </ChildPackages>
-          </Package>
-        </ChildPackages>
-      </Package>
-      <!--Adds more projections and geotransformations.-->
-      <Package name="Additional Projection Engine Transformations" enabled="false" />
-    </ChildPackages>
-  </Package>
-  <!--Enables debugging options within the runtime.-->
-  <!--Debugging options should only be used for testing purposes, and not deployed in final solutions.-->
-  <Package name="Debug" enabled="false">
-    <ChildPackages>
-      <!--Adds support for runtime logging.-->
-      <Package name="Logging" enabled="false" />
-    </ChildPackages>
-  </Package>
-</Packages>
```

Comment:
Keeping this deployment file would allow for switching between Local Server v100.3 and Local Server v100.2. Unlikely to be a requirement in the context of the samples, but worth considering before removing.